compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/AwtWindow.desktop.kt

```diff
@@ -81,9 +80,6 @@ fun <T : Window> AwtWindow(
     UpdateEffect {
         val window = window()
         update(window)
-        if (!window.isDisplayable) {
```

Comment:
I don't remember why, but probably it is because initially it was added only here.

Anyway, I agree, it should be removed from AwtWindow, as it doesn't contain Compose content